### PR TITLE
Update run-action CLI text

### PR DESCRIPTION
**Pull request by timClicks**
Grammar/style improvements. Make style consistent with other docs.
Explain --wait option. Add explanations for each of the examples. Link
to web documentation.

## QA steps

* Run `juju help run-action`

## Documentation changes

-

## Bug reference

* https://bugs.launchpad.net/juju/+bug/1835962

### CODE REVIEW

File: cmd/juju/action/run.go [hunk 1]
```diff
@@ -51,55 +51,87 @@ type runCommand struct {
...
 
 If the leader syntax is used, the leader unit for the application will be
 resolved before the action is enqueued.
 
-Params are validated according to the charm for the unit's application.  The
-valid params can be seen using "juju actions <application> --schema".
-Params may be in a yaml file which is passed with the --params option, or they
-may be specified by a key.key.key...=value format (see examples below.)
+Many actions take parameters. Params can be supplied as a YAML file or as 
+arguments on the command line. Specify a file with the --params <params.yaml>.
```
Comment (hpidcock) [inline cmd/juju/action/run.go]: `Specify a file with --params <params.yaml>.`

**Comment by timClicks** [comment]
!!build!!

**Comment by timClicks** [comment]
!!build!!

**Comment by timClicks** [comment]
$$merge$$

**Comment by timClicks** [comment]
Closing this PR as run-action semantics is undergoing significant changes. We can reassess then.

**Pull request closed without merging**